devfiles/che4z/devfile.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: 1.0.0
+metadata:
+  generateName: che4z-
+components:
+  - 
+    type: chePlugin
+    id: broadcommfd/cobol-language-support/latest
+  - 
+    type: chePlugin
+    id: bitlang/cobol/latest
+  - 
+    type: chePlugin
+    id: broadcommfd/explorer-for-endevor/latest
+  - 
+    id: eclipse/che-machine-exec-plugin/7.3.0
+    type: chePlugin
+  - 
+    id: eclipse/che-theia/7.3.0
+    type: cheEditor
```

Comment:
Devfiles should not contain specific versions of `che-theia` and `che-machine-exec-plugin`, since that overrides Che automatically using the latest release (see [che.properties](https://github.com/eclipse/che/blob/6e0ce6d101b337e0d77ffa1b25f5b0bcd621ce06/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L558-L571))

Reply:
We are aware of this, but we prefer to release with version we tested against. It happened in the past that our extension stopped working in next release of che-theia, for example because version of Theia framework changed. This does not affect user's ability to switch che-theia to 'latest' in workspace configuration.

Reply:
That makes sense to me :+1: